app/src/main/res/layout/logins_item.xml

```diff
@@ -51,16 +53,13 @@
         android:id="@+id/usernameView"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
-        android:layout_marginEnd="12dp"
-        android:layout_weight="1"
-        android:ellipsize="middle"
```

Comment:
I think this should still be kept?
Didn't see anything related to this in the specs..

Reply:
They were looking unnecessary to me, i can re-add them.

Reply:
I was referring just to `android:ellipsize="middle"`.
Otherwise we'd ellipsize at the end. 
If nothing changed specifically about this in the specs we should keep previous behavior.

Reply:
Ok, added ellipsize and weight back. They currently stays as before.